kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/schemaextractor/ExtractSchemaRule.kt

```diff
@@ -220,13 +220,24 @@ class SchemaExtractor(
       val pathArg = requireNotNull(argumentLists.second.single().getArgumentExpression()?.text?.trim { it == '\"' }) {
         "$sourcePos Could not extract path from ${verb.name} @Ingress annotation"
       }
+
       MetadataIngress(
-        path = pathArg,
+        path = extractPathComponents(pathArg),
```

Comment:
@worstell Just a heads up. I had to update this bit to support the new path components. Lemme know if you want any changes here! :)

Reply:
lgtm!